includes/class-move.php

```diff
@@ -83,7 +83,7 @@ private static function update_user_also_known_as( $user_id, $from ) {
 	 * @param string $from The current account URL.
 	 */
 	private static function update_blog_also_known_as( $from ) {
-		$also_known_as   = (array) \get_option( 'activitypub_blog_user_also_known_as' );
+		$also_known_as   = \get_option( 'activitypub_blog_user_also_known_as' ) ?? array();
```

Comment:
Would passing an empty array as the default work here?

Reply:
the default is only used if the option was never set before. so if it is set to null or an empty string, it will not return the fallback.

Reply:
> $default_value - Default value to return if the option does not exist.

https://developer.wordpress.org/reference/functions/get_option/#parameters